op-proposer/bindings/l2outputoracle.go

```diff
@@ -26,8 +26,16 @@ var (
 	_ = common.Big1
 	_ = types.BloomLookup
 	_ = event.NewSubscription
+	_ = abi.ConvertType
```

Comment:
what's this?

Reply:
it is used for the custom type in Solidity, e.g. `EOTSInfo`